### DISCUSSION TITLE
unknown blob error when i'm trying to push image to Github Package Registry by Actions

### DISCUSSION THREAD

**Discussion by webmakaka** [community/community#25822] [Actions]
<p>Hello experts!</p>
<p>I study Github Actions + Github Package Registry.</p>
<p>I have an error when i’m trying to push image to Github Package Registry by Actions</p>
<pre><code>Login Succeeded
The push refers to repository [docker.pkg.github.com/marley-nodejs/mern-stack-front-to-back-v2.0/client.anketa.info]
412bcb172467: Preparing
5d7983547f5b: Preparing
ce3539cc1849: Preparing
16d1b1dd2a23: Preparing
2db44bce66cd: Preparing
unknown blob
##[error]Process completed with exit code 1.
</code></pre>
<p>Here is link on error:</p>
<p><a href="https://github.com/marley-nodejs/MERN-Stack-Front-To-Back-v2.0/runs/221735767" rel="nofollow noopener">https://github.com/marley-nodejs/MERN-Stack-Front-To-Back-v2.0/runs/221735767</a></p>
<p>Project is opensource, you can check everything.</p>
<p>What is wrong?</p>
<p>Any suggestions?</p>
<p>If i made this topic in not right place, please move it.</p>
<p>Thanks.</p>

**Comment by webmakaka** [comment]
<p>I tried example by <a href="https://github.com/chrispat/actions-container-service" rel="nofollow noopener">Chris Patterson</a>.</p>
<p>And this example works for me.</p>
<p><a href="https://github.com/marley-nodejs/actions-container-service/commit/061a67692d24dce75a4cf970d00f70ed831fd52b/checks" rel="nofollow noopener">https://github.com/marley-nodejs/actions-container-service/commit/061a67692d24dce75a4cf970d00f70ed831fd52b/checks</a></p>
<p>==========</p>
<p>But i still did not solve problem with my project.</p>

**Comment by collinbarrett** [comment]
<p>I think it might be an issue with Package Registry. I’m seeing the same thing. I reported to GitHub Support, and they seemed to confirm an issue on their end escalated to engineering.</p>
<p>My Example:</p>
<p>It looks like I am getting some new assets on my package page every time I try to push, but the only version that shows is “docker-base-layer” when I should see tags for “latest” and “[MyBuildId]”. <a href="https://github.com/collinbarrett/FilterLists/packages/26496" rel="nofollow noopener">https://github.com/collinbarrett/FilterLists/packages/26496</a></p>
<p>Build Def: <a href="https://github.com/collinbarrett/FilterLists/blob/1ed56d3bf3e75e1a6e80993f370a8126067afe3a/azure-pipelines.web.yaml" rel="nofollow noopener">https://github.com/collinbarrett/FilterLists/blob/1ed56d3bf3e75e1a6e80993f370a8126067afe3a/azure-pipelines.web.yaml</a><br>
Build Results: <a href="https://dev.azure.com/collinbarrett/FilterLists/_build/results?buildId=1561" rel="nofollow noopener">https://dev.azure.com/collinbarrett/FilterLists/_build/results?buildId=1561</a></p>

**Comment by laurentkempe** [comment]
<p>I have exactly the same issue with Github Package Repository than <a class="mention" href="/collinbarrett">@collinbarrett</a> while pushing, only “docker-base-layer” is updated! So it seems that this issue is still not fixed</p>

**Comment by hoppergee** [comment]
<p>I have also met this issue. But after I run the task again, everything works fine. Seems it’s a random issue.</p>

**Comment by webmakaka** [comment, marked as answer]
<p>Today i received an email “Announcing general availability for GitHub Actions and GitHub Package Registry”.</p>
<p>Clicked here  <a href="https://github.com/features/package-registry" rel="nofollow noopener">https://github.com/features/package-registry</a></p>
<p>On “SignUp for Free Beta”</p>
<p>And tried run with my configs from post 1.</p>
<p>Looks everything works.</p>
<p><a href="https://github.com/marley-nodejs/MERN-Stack-Front-To-Back-v2.0/actions" rel="nofollow noopener">https://github.com/marley-nodejs/MERN-Stack-Front-To-Back-v2.0/actions</a></p>
<p>Thanks</p>

**Comment by laurentkempe** [comment]
<p>Yeah <a class="mention" href="/webmakaka">@webmakaka</a> the workflows runs! But, if you get back from the Github Package Registry your just built docker image with the tag latest, which is the one you use to build, do you get the last change? I don’t, it seems that it is the tag docker-base-layer which gets the changes</p>

**Comment by laurentkempe** [comment]
<p>Forget my last message! It seems that it is working now, at least when I get back the latest image on my local machine, the image got my latest code change. So, it seems that it is my CD action which doesn’t work</p>

**Comment by webmakaka** [comment]
<p>Hi <a class="mention" href="/laurentkempe">@laurentkempe</a> l</p>
<p>I made another one question about GPR.</p>
<p><a href="https://github.community/t/error-response-from-daemon-unauthorized-when-im-trying-to-pull-public-image-from-gpr/2614">https://github.community/t5/How-to-use-Git-and-GitHub/Error-response-from-daemon-unauthorized-when-I-m-trying-to-pull/m-p/36323</a></p>
<p>Possible you know solution.</p>
<p>If know please answear.</p>
<p>Thanks</p>
